### ISSUE TITLE
Stale-content flag

### ISSUE THREAD

**Issue by iamstevebeatty** [Automattic/newspack-theme#947]
Hoped for addition to Newspack: a prominent flag at the top of an older story that says that the story is more than X months/years old. This will help in the fight against disinformation. 

Ideally, we offer the controls to globally let the publisher set the age of of a post that will include the flag. 

Here's how the Tucson Sentinel handles it:
![Screen Shot 2020-05-22 at 10 08 40 AM](https://user-images.githubusercontent.com/48489469/82682573-15ab1000-9c15-11ea-90de-1149b7c3f338.png)

The Tucson Sentinel also has logic that puts the year on an overlay of an image, for social media and other linked postings:
![image](https://user-images.githubusercontent.com/48489469/82683126-0d9fa000-9c16-11ea-8fc5-7cb1c9271cc5.png)

And The Guardian:
![Screen Shot 2020-05-22 at 10 20 11 AM](https://user-images.githubusercontent.com/48489469/82683034-e3e67900-9c15-11ea-8b41-51d6d9171b72.png)

**Comment by philipjohn** [comment]
Came here to suggest this after seeing that [Facebook are beginning to warn people](https://about.fb.com/news/2020/06/more-context-for-news-articles-and-other-content/amp/?__twitter_impression=true) before sharing anything over 3 months old.

**Comment by onthewight** [comment]
Big supporter of this. A read help to the reader to understand that it's not current. People read Website so quickly these days - this can really help avoiding old news being mistaken for current.

**Comment by eecoston** [comment]
I agree this would be a helpful function. If possible, could it also add the year to the social images, similar to how The Guardian does it?

**Comment by tmmbecker** [comment]
@laurelfulford I'm resurrecting this one as HKFP, SMC and Scalawag have all asked about it (in the same thread). As per our convo, we'll need to figure out how to add this and make sure that folks know how this feature differs from the existing time-ago feature in the Customizer settings.
